proprietary/Components/src/denhaag/responsive-content.tokens.json

```diff
@@ -0,0 +1,19 @@
+{
+  "denhaag": {
+    "responsive-content": {
+      "max-width": { "value": "calc(100vw - (var(--denhaag-space-inline-xl)))" },
```

Comment:
Unfortunately this `calc` is not something that is supported outside of `css` and therefor not a great thing to use in the Style Dictionary design tokens. So we should use the actual value instead :(

Reply:
I've removed the calc function and added a border-box alternative with a padding design token.